ramin/data_type/data_type.js

```diff
@@ -0,0 +1,148 @@
+//number
+
+let num1 = 10;
```

Comment:
you should always use `const` unless you really need to reassign.
https://stackoverflow.com/questions/41086633/in-javascript-why-should-i-usually-prefer-const-to-let

Reply:
okay sure